src/model.py

```diff
@@ -496,21 +513,58 @@ def _create_response(
                 )
             )
 
+        # logprobs
```

Comment:
nit: Let's try to provide more descriptive comments, i.e. "Handling logprobs outputs (+ link on vllm doc)"

Reply:
It is a good idea to add the link, so it is easier to lookup the vLLM implementation in the future.

I think it would be redundant to add "handling" and "outputs" to the comment, because the method name "_create_response" already imply handling outputs, and the only missing information here is which output is this block/section is handling, which I set the comment to match the output tensor name exactly to be very clear that this block is handling logprobs outputs. What do you think?

Reply:
Sure, I'm ok with just adding link.